functional/iak-idevid-register-with-certificates/test.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+HTTP_SERVER_PORT=8080
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+TPM2_OPENSSL="https://github.com/tpm2-software/tpm2-openssl/releases/download/1.2.0/tpm2-openssl-1.2.0.tar.gz"
+CA_PWORD="keylimeca"
+CERT_DIR="/var/lib/keylime"
+TPM_CERTS="/var/lib/keylime/tpm_cert_store"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
```

Comment:
Could you please create a temporary directory in the setup phase and do the execution inside this directory (leaving and removing it in the cleanup phase)? For example check https://github.com/RedHat-SP-Security/keylime-tests/blob/main/functional/tenant-runtime-policy-sanity/test.sh#L15
We do not have it in all tests but it is better to avoid creating additional artifacts in the test directory itself.